### PR TITLE
gtk-doc, bump version

### PR DESCRIPTION
**Pull request by Begasus**
WIP ... the *.pc is still not in the correct place ...

### CODE REVIEW

**Comment by korli** [comment]
I had a look at what arch Linux does, the pc file is in share/pkgconfig, which would translate to $dataDir/pkgconfig on Haiku. I'm not sure that pkg-config picks up this path though. To be tested.

**Comment by Begasus** [comment]
you are right there, noticed it already yesterday (hence wip), can't seem to create a devel package for it so looking into mv it to the correct place ...

**Comment by pulkomandy** [comment]
Moving it should be fine. I'm not in Haiku at the moment but it should be possible to ask pkg-config to print its search dir. However, having this in share makes it difficult to handle hybrid arch properly so it would be nice to avoid it.

**Comment by korli** [comment]
Why is it needed to have secondary arches?

**Comment by Begasus** [comment]
jikes seem to have messed some things up :/

File: dev-util/gtk_doc/gtk_doc-1.25.recipe [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	#TODO fix installation path so it installs 
+	#in the correct directory for each arch
+	# Remove stuff we don't need in the secondary architecture base package.
+	if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	mkdir -p $developDir/lib/x86/pkgconfig/
+	mv $dataDir/pkgconfig/gtk-doc.pc $developDir/lib/x86/pkgconfig/gtk-doc.pc
+	else
+	mkdir -p $developDir/lib/pkgconfig/
+	mv $dataDir/pkgconfig/gtk-doc.pc $developDir/lib/pkgconfig/gtk-doc.pc
+	fi
```
Comment (Begasus) [inline dev-util/gtk_doc/gtk_doc-1.25.recipe]: @korli this is the place where I moved the *.pc file to it's correct place, it's probably not the right way, but this is how far I came so far

**Comment by Begasus** [comment]
@korli secondary arches are needed because they don't seem to pick up the *.pc file otherwise (tested with rasqal)

**Comment by korli** [comment]
Please add a TODO where the pkgconfig files should be located. pkgconfig needs to be fixed to pick up files  in a architecture independent path aka $dataDir/pkgconfig.

**Comment by Begasus** [comment]
in the recipe just above the section I already added a TODO ...

**Comment by korli** [comment]
Well the TODO you inserted means something else to me. Whatever...

**Comment by Begasus** [comment]
'proper' fix coming up, no need to check for the arch's anymore (just need to do some tests)

**Comment by Begasus** [comment]
updated the recipe and added patch, hope this makes more sense :)

**Comment by korli** [comment]
I don't know, it can always be improved later if needed.

**Pull request merged**